pkg/rpctype/rpctype.go

```diff
@@ -18,6 +18,11 @@ type Input struct {
 	Prog   []byte
 	Signal signal.Serial
 	Cover  []uint32
+	// We'd like to record raw coverage each time the input appears.
+	// So the type signature is a bit complicated here:
+	// call id * part id -> cover (which is itself an array)
+	// call id 0 is for extra coverage, then come normal calls within that program.
+	RawCoverParts [][][]uint32
```

Comment:
It looks ugly here. What's passed over RPC is really just `[]uint32`. How hard is it to switch syz-manager to not use this RPC struct as corpus elements?

Reply:
Not so hard, separated those structures.